### PR TITLE
Adding BF status update

### PR DESCRIPTION
**Pull request by hflynn**
Edited and formatted version of https://docs.google.com/document/d/1dBxM6qVuZ0--fi573B8FTbKhBaK0G8Qn4nwFICCRCJA/edit#

No staging pages, merging will publish!

Can be built locally with jekyll but I already checked the formatting so it mostly just needs another proof read to make sure I haven't missed anything.

### CODE REVIEW

File: _posts/2015-01-13-bf-update.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+   critical components of metadata (e.g. experimental protocol metadata)
+   stored in OME-TIFF or OMERO or read by Bio-Formats.
+
+*  Changing channel wavelength from 'int' to 'float' - this is a long overdue
+   fix which recognizes the increasing use of illumination systems with
+   special wavelengths that must be represented with better than integer
+   accuracy.
+
+*  Unit support - this is by far the most substantial change in the 5.1.x
+   series. We want as many of the metadata values stored in OME formats to be
```
Comment (mtbc) [inline _posts/2015-01-13-bf-update.md]: "as many" ... but no following "as"

File: _posts/2015-01-13-bf-update.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+   the importance of the units is that recording the pixel size of the images
+   in these different domains is critical for proper measurements and
+   analysis. In the longer term, we aim to record and express the relationship
+   between different images, e.g. in correlative imaging, so we must have this
+   capability.
+
+###C++ implementation
+
+For OME 5.1 we are releasing a native C++ implementation for Bio-Formats. The
+goal here is to make it easier to call Bio-Formats from a non-Java programme.
```
Comment (mtbc) [inline _posts/2015-01-13-bf-update.md]: British spelling.

File: _posts/2015-01-13-bf-update.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+The first versions of this work will be released with 5.1.0 but the API will
```
Comment (mtbc) [inline _posts/2015-01-13-bf-update.md]: "versions" is plural?

File: _posts/2015-01-13-bf-update.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+be subject to change until the 5.2. release later in 2015. Our goal is again
```
Comment (mtbc) [inline _posts/2015-01-13-bf-update.md]: "the 5.2" reads oddly

File: _posts/2015-01-13-bf-update.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+In our upcoming 5.1.x releases, we are instituting a series of model changes
+that substantially expand OME’s capabilities. Some of these simply reflect
+corrections or additions that are necessary to keep the software fresh and
+up-to-date. Others are additions that we intend to be the foundation for work
+in the next 1-2 years.
+
+Examples include:
+
+*  Map annotation - this is an metadata feature that allow users to use
+   annotations which take the form of  key value pairs (e.g. ‘cell line’ and
```
Comment (mtbc) [inline _posts/2015-01-13-bf-update.md]: double space

File: _posts/2015-01-13-bf-update.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+several HCS readers to have faster initialization and plane reading times.
+
+##Looking a bit further forward...
+
+After the 5.1. release, we of course have several more things we are
+considering. To get changes to Bio-Formats out to the community as rapidly as
+possible, we will begin the process of decoupling Bio-Formats releases from
+the rest of the OME project’s work. We have released all the different tools
+together because, in fact, they are closely related, and a release process
+requires a lot of the same testing and process (release testing
```
Comment (mtbc) [inline _posts/2015-01-13-bf-update.md]: hyphenate "release-testing" verb phrase

File: _posts/2015-01-13-bf-update.md (hunk 1 above)
Comment (hflynn) [inline _posts/2015-01-13-bf-update.md]: it's 'as many of' that's why

**Comment by jburel** [comment]
Few comments:

- ``an metadata feature``` => a
- ```Map annotation```: Term is unfortunate, since we allow duplicate key
do we want to highlight the fact that we can have duplicate key?
- ```where Python and C++ dominate```. Do we need to indicate Python since we are talking about C++
 - ```e.g., ``` comma not needed

File: _posts/2015-01-13-bf-update.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+be subject to change until the 5.2 release later in 2015. Our goal is again
+to expand the number of domains in which OME software can be used, in
+particular into applications such as ITK, OpenCV, and NumPy in medical
+imaging.
+
+For the moment, our work here is related to basic I/O functionality
+(reading/writing OME-TIFF etc.). This proves the concept and allows us to look
+forward to adding specific readers that will benefit the community. 
+
+*Disclaimer: not all, or even most of the Java-based Bio-Formats will be
```
Comment (ximenesuk) [inline _posts/2015-01-13-bf-update.md]: Strictly there should be a comma after 'most'

File: _posts/2015-01-13-bf-update.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+ported to C++.*
+
+###Format fixes
+
+As always we have been working on fixing several formats. This work has been
+complicated by the number of imaging modalities which are appearing inside in
+each individual file format (see [previous blog post](http://blog.openmicroscopy.org/file-formats/2014/10/16/file-formats/)).
+Our work is focused on ND2, MetaMorph, Prairie, DICOM, Zeiss CZI, and Leica
```
Comment (ximenesuk) [inline _posts/2015-01-13-bf-update.md]: Ah, an Oxford comma :-)

File: _posts/2015-01-13-bf-update.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+LIF. We also have added support for the i2i and im3 formats.
+
+###Performance
+
+Dataset sizes continue to increase by several-fold, so Bio-Formats’
+capabilities have to keep up. For the 5.1. release, we’ve reduced the
```
Comment (ximenesuk) [inline _posts/2015-01-13-bf-update.md]: Should it be '5.1' rather than '5.1.' ?

File: _posts/2015-01-13-bf-update.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+overhead for metadata parsing, improved I/O buffering, and tuned Zeiss CZI and
+several HCS readers to have faster initialization and plane reading times.
+
+##Looking a bit further forward...
+
+After the 5.1. release, we of course have several more things we are
```
Comment (ximenesuk) [inline _posts/2015-01-13-bf-update.md]: 5.1. again.

**Comment by joshmoore** [comment]
:+1: from me.

**Pull request merged**